pandas/core/base.py

```diff
@@ -50,6 +50,8 @@
 class PandasObject(DirNamesMixin):
     """baseclass for various pandas objects"""
 
+    _deprecations = DirNamesMixin._deprecations | frozenset()  # type: FrozenSet[str]
```

Comment:
Is the rhs of the union doing anything here? I assume you are just stubbing out for future readers but in that case the annotation is unnecessary. 

Probably cleanest to just leave as is for now and can be overridden in future as required

Reply:
Yeah, it's for the readers. I don't think this pattern of copying frozensets is seen very much, so thought it helps a bit to just have it stubbed out.